app/src/main/java/com/globant/practice/presentation/view/fragment/SubscriberListFragment.java

```diff
@@ -68,6 +68,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle(R.string.large_app_name);
+        ((AppCompatActivity) getActivity()).getSupportActionBar().setDisplayHomeAsUpEnabled(false);
```

Comment:
I need this because the first time when the fragment is showing the value is false but when change to other fragment the other fragment set the value to true and when go back to the SubscriberListFragment they will show the arrow to go back because the ActionBar is the same for all fragments.